### DISCUSSION TITLE
Using custom buttons in Layout Header

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2198] [Help] [answered]
Hi, 
Is there a way I can put a button in the layout header? or maybe reprogram the apps button in the top left? I'm looking for a quick way to log a user out.
![image](https://user-images.githubusercontent.com/80058238/127689778-c1a8273c-9696-45da-8e6e-1dbb0ff42aa1.png)

**Comment by shogunpurple** [comment, marked as answer]
Hi @AndrewJRamay - this is not possible with the standard nav layout. You would either have to use the empty layout with your own navbar and logout actions, or stick with using the apps button on the top right to take you back to the portal to log out.

**Reply by KernOil** [reply]
@shogunpurple  ok thank you.
